scripts/all-tests.sh

```diff
@@ -12,7 +12,7 @@ set -e
 
 # Violations thresholds for failing the build
 export PYLINT_THRESHOLD=4175
-export JSHINT_THRESHOLD=7550
+export ESLINT_THRESHOLD=64345
```

Comment:
Not a typo; we are allowing ~10x more violations initially. This is due to the fact that we're being much, much stricter about our JavaScript style with the adoption of the new styleguide.

Forthcoming PRs will reduce this by about half (autofixing) and possibly more (by adding repo-specific rule exceptions based on our analysis of the most commonly violated rules in existing code).